scripts/install-helpers/baremetal-coco/README.md

```diff
@@ -101,30 +101,38 @@ The deployment sequence is described below:
 ## Install OSC operator pre-GA release
 
 - Update osc_catalog.yaml to point to the pre-GA catalog
-  For example if you want to install the pre-GA 0.0.1-24 build, then change the
-  image entry to the following
+
+  Use the following command to find out the latest catalog
 
   ```sh
-  image: quay.io/openshift_sandboxed_containers/openshift-sandboxed-containers-operator-catalog:0.0.1-22
+  curl -s 'https://quay.io/api/v1/repository/redhat-user-workloads/ose-osc-tenant/osc-test-fbc/tag/?limit=1000' | jq -r '.tags[] | select(.name | test("^osc-test-fbc-on-push-.*-build-image-index$")) | "\(.start_ts) \(.name)"' | sort -nr | head -n1 | awk '{print "quay.io/redhat-user-workloads/ose-osc-tenant/osc-test-fbc:" $2}
   ```
 
-- The pre-GA build images are in an authenticated registry, so you'll need to
-  set the `PULL_SECRET_JSON` variable with the registry credentials. Following is an example:
+  Replace the `image:` entry with the returned value. For example:
 
   ```sh
-  export PULL_SECRET_JSON='{"brew.registry.redhat.io": {"auth": "abcd1234"}, "registry.redhat.io": {"auth": "abcd1234"}}'
+  image: quay.io/redhat-user-workloads/ose-osc-tenant/osc-test-fbc:osc-test-fbc-on-push-cldfq-build-image-index
   ```
 
 - Kickstart the installation by running the following:
 
   ```sh
-  ./install.sh -t tdx -m -s -b
+  ./install.sh -t tdx -m -b
   ```
 
   This will deploy the pre-GA release of OSC operator on TDX hosts.
 
 After successful install `kata`, `kata-tdx` or `kata-snp` runtimeclasses will be created based on the TEE type.
 
+## Adding custom container registry secrets
+
+  To add custom registry secrets, you'll need to set the `PULL_SECRET_JSON` variable with the registry credentials.
+  Following is an example:
+
+  ```sh
+  export PULL_SECRET_JSON='{"brew.registry.redhat.io": {"auth": "abcd1234"}, "registry.redhat.io": {"auth": "abcd1234"}}'
```

Comment:
AFAIK we're not to pull from brew ever again and the images we push to quay.io are expected to be publicly accessible... do we really need `PULL_SECRET_JSON` ?